server/Dockerfile

```diff
@@ -1,27 +1,30 @@
 # dev build
 FROM ghcr.io/immich-app/base-server-dev:202505131114@sha256:cf4507bbbf307e9b6d8ee9418993321f2b85867da8ce14d0a20ccaf9574cb995 AS dev
 
-RUN apt-get install --no-install-recommends -yqq tini
+RUN apt-get install --no-install-recommends -yqq tini && \
+  rm -rf /usr/src/app && \
```

Comment:
There a bunch of leftover stuff in here from the build of the dev-base-image that needs to be cleaned up before proceeding, otherwise its just a little confusing.